std/io/bufio_test.ts

```diff
@@ -447,21 +447,24 @@ Deno.test("readStringDelimAndLines", async function (): Promise<void> {
   assertEquals(lines_, ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"]);
 });
 
-Deno.test(async function bufReaderShouldNotShareArrayBufferAcrossReads() {
-  const decoder = new TextDecoder();
-  const data = "abcdefghijklmnopqrstuvwxyz";
-  const bufSize = 25;
-  const b = new BufReader(stringsReader(data), bufSize);
-
-  const r1 = (await b.readLine()) as ReadLineResult;
-  assertNotEOF(r1);
-  assertEquals(decoder.decode(r1.line), "abcdefghijklmnopqrstuvwxy");
-
-  const r2 = (await b.readLine()) as ReadLineResult;
-  assertNotEOF(r2);
-  assertEquals(decoder.decode(r2.line), "z");
-  assert(
-    r1.line.buffer !== r2.line.buffer,
-    "array buffer should not be shared across reads"
-  );
-});
+Deno.test(
+  "bufReaderShouldNotShareArrayBufferAcrossReads",
+  async function (): Promise<void> {
```

Comment:
we should use arrow syntax when providing a function as a parameter

Reply:
This syntax is not enforced at the moment anyway and there's plenty of tests using `async function (): Promise<void>`.